src/util/MongoBinary.js

```diff
@@ -148,6 +148,21 @@ export default class MongoBinary {
 
     if (systemBinary) {
       binaryPath = await this.getSystemPath(systemBinary);
+      const binaryVersion = execSync('mongod --version')
```

Comment:
Should be
```js
const binaryVersion = execSync(`${binaryPath} --version`)
```

And need to add check that `binaryPath` is not empty ;)

Reply:
oh yeah, good catch totally missed that.

Reply:
I've just merged with the master (there was a conflict).
Pull changes before commit.

Reply:
There were updated packages in master.
Maybe it helps to resolve #120

Reply:
👌 I made that fix and pushed.